src/components/ADempiere/Form/VPOS/Collection/index.vue

```diff
@@ -495,6 +505,9 @@ export default {
     },
     fieldsPaymentType() {
       return this.fieldsList[2]
+    },
+    overInvoice() {
```

Comment:
Change this by overUnderPayment

Reply:
Hello, @yamelsenih  thank you for your comment, the computed property has been renamed.